src/components/Section.jsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react"
+import styled from "styled-components"
+
+import { Grid } from "../components/Grid"
+
+const Container = styled.div`
+  background-color: var(--color-heading);
+  padding: var(--spacing-24);
+  display: grid;
+  gap: 5rem;
```

Comment:
Can you please explain why the `Container` component is `grid` with a `gap`? It seems to have only 1 child: `<Grid/>`. Perhaps this can be removed?

Reply:
Yes, you are right. I will delete grid and gap properties.
It was the result of the experimenting in the process. I missed unneeded properties.